src/qtt/algorithms/random_telegraph_signal.py

```diff
@@ -260,7 +261,7 @@ def tunnelrates_RTS(data: Union[np.ndarray, qcodes.data.data_set.DataSet], sampl
 
     # binning the data and determining the bincentres
     if num_bins is None:
-        num_bins = int(np.sqrt(len(data)))
+        num_bins = min(int(np.sqrt(len(data))), 1200)
```

Comment:
earlier, creating the figure, data array was binned to a max of 800, now 1200?

Reply:
The binning is different. In this fist case it is for plotting a 2D histogram, for the second case for fitting data. I made the 1200 an explicit variable to clarify. The exact amounts are not really important for these high numbers though.